src/asciidoctorFindIncludeProcessor.ts

```diff
@@ -0,0 +1,49 @@
+interface includeEntry {
```

Comment:
By convention, interface and class name should begin with an upper letter, so `IncludeEntry`.
`includeEntry` could be the name of a variable.

Reply:
Very good. It shall be so.